File: 1217-minimum-cost-to-move-chips-to-the-same-position/1217-minimum-cost-to-move-chips-to-the-same-position.java
class Solution {
    public int minCostToMoveChips(int[] position) {
        int evenCount = 0, oddCount = 0;
        for(int num: position){
            if(num%2 == 0) evenCount++;
            else oddCount++;
        }
        
        return (oddCount <= evenCount) ? oddCount : evenCount;
    }
}

File: 1217-minimum-cost-to-move-chips-to-the-same-position/1217-minimum-cost-to-move-chips-to-the-same-position.js
/**
 * @param {number[]} position
 * @return {number}
 */
var minCostToMoveChips = function(position) {
    let evenCount = 0, oddCount = 0;
    for(let num of position){
        if(num%2 == 0){
            evenCount++;
        }else{
            oddCount++;
        }
    }
    return Math.min(evenCount,oddCount);
};